PRS8001-9000/PRS8422Salomon.xml

```diff
@@ -48,46 +48,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sameAs="wd:Q343719" sex="1">
-
-
+            <person sameAs="wd:Q717219" sex="1">
                   <persName xml:lang="gez" xml:id="n1" type="birth">ሰሎሞን፡</persName>
                   <persName xml:lang="gez" corresp="#n1" type="normalized">Salomon II</persName>
```

Comment:
I wonder if "II" should be marked up as addName
even `<addName>II</addName> (or <addName>III</addName>)`